### PR TITLE
Update tubemq_user_guide.md

### PR DESCRIPTION
**Pull request by JunpingDu**
Make some minor changes to document, fix some issues of grammar and expression.

This issue get reported via: https://github.com/Tencent/TubeMQ/issues/60

### CODE REVIEW

File: docs/tubemq_user_guide.md [hunk 1]
```diff
@@ -127,17 +127,17 @@ zkCommitFailRetries=10
...
+server IPs in this way, assume the ip is `192.168.1.2`:
 ##### /etc/hosts
 ```
 192.168.1.2 192-168-1-2
 ```
 
 ## High Availability
 
-In the example, we run all the service on a single node. In the real production environment, you may
+In above example, we run the services on a single node. However, in the real production environment, you may
```
Comment (tisonkun) [inline docs/tubemq_user_guide.md]:
```suggestion
In the example above, we run the services on a single node. However, in real production environment, you
```

File: docs/tubemq_user_guide.md [hunk 2]
```diff
@@ -127,17 +127,17 @@ zkCommitFailRetries=10
...
 need to run multiple master services on different servers for high availability purpose. Here's
```
Comment (TisonKun) [inline docs/tubemq_user_guide.md]:
```suggestion
need to run multiple master services on different servers to enable high availability. Here's
```

File: docs/tubemq_user_guide.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # TubeMQ User Guide
 ## Prerequisites
 
-- Java 7/8(Java 9+ are not test)
+- Java 7/8(Java 9+ are not verified yet)
```
Comment (tisonkun) [inline docs/tubemq_user_guide.md]:
```suggestion
- Java 1.7 or 1.8(Java 9 and above haven't been verified yet)
```

**Review by tisonkun — CHANGES_REQUESTED** [review]
Thanks for your contribution @JunpingDu! I left several inline comments.

**Review by JunpingDu — COMMENTED** [review]
Thanks for comment, Tison. Latest PR should address your comments.

**Review by tisonkun — APPROVED** [review]
Thanks for your updates @JunpingDu ! LTGM.

Merging...

**Pull request merged**